projects/cryptoChampz.md

```diff
@@ -0,0 +1,75 @@
+## Project Name
+
+"Crypto Champz"
+
+## Category
+
+NFT Gaming
+
+## Project Description
+
+Crypto Champz is a Universal Character Generator that takes advantage of NFT composability to create robust characters that are playable in an endless variety of games. We use Chainlink VRF to randomly generate Champion characteristics such as Hometown, Background, Skills, Traits, Stats (Strength, Charisma, etc.) and more! Each Champion also has artwork that was custom designed for our project, and there are multiple variations of artwork available. Putting it all together, there are over 100 trillion different combinations, ensuring each Champion is special.
+
+Crypto Champz serve as NFT primitives that can be used to traverse the Metaverse of blockchain games. For example, your Champion might be used in a card game, a virtual board game, a farming simulator, or even your next D&D campaign! Of course, it can also be traded as a collectible. To jump start the ecosystem, we have created two mini games. One game lets you battle your Champz with other players. Chainlink oracles are used to retrieve weather conditions which can affect the outcome of the duel. Another game has a DeFi twist with Chainlink Price Feeds - players get rewarded for loyalty to high performing assets.
+
+## Use of IPFS, Filecoin and Libp2p
+
+We use IPFS to store the images for our NFTs.
+
+## Project Status
+
+Under development
+
+## Previews
+
+https://i.imgur.com/vwOKoIg.png
```

Comment:
Can you format so it shows up as an image?

Reply:
Formatted this as an image.